sdk/examples/how_tos/alias/create.rs

```diff
@@ -44,7 +44,7 @@ async fn main() -> Result<()> {
         .retry_transaction_until_included(&transaction.transaction_id, None, None)
         .await?;
     println!(
-        "Block included: {}/block/{}",
+        "Transaction included: {}/block/{}",
```

Comment:
This should stay Block (we have that 57 times, `Transaction included` only 3 times)

Reply:
Ok will replace the rest to `Transaction` then.

Reply:
But the link has the block and not transaction